pandas/io/formats/css.py

```diff
@@ -75,101 +130,79 @@ def __call__(self, declarations_str, inherited=None):
         props = dict(self.atomize(self.parse(declarations_str)))
         if inherited is None:
             inherited = {}
+        assert inherited is not None
+
+        props = self._update_initial(props, inherited)
+        props = self._update_font_size(props, inherited)
+        return self._update_other_units(props)
 
+    def _update_initial(
+        self,
+        props: Dict[str, str],
+        inherited: Dict[str, str],
+    ) -> Dict[str, str]:
         # 1. resolve inherited, initial
         for prop, val in inherited.items():
             if prop not in props:
                 props[prop] = val
 
-        for prop, val in list(props.items()):
+        for prop, val in props.copy().items():
```

Comment:
don't copy in the interable, rather copy *first* and iterate over the original.

Reply:
Changed it. But, out of curiosity, why would it matter?

Reply:
doesn't matter, just style-wise its easier to read, e.g. less things then going on in the iteration part